packages/@aws-cdk/aws-stepfunctions-tasks/lib/sagemaker-task-base-types.ts

```diff
@@ -626,6 +626,181 @@ export enum AssembleWith {
 
 }
 
+/**
+ * Specifies the containers in the inference pipeline.
+ *
+ * @experimental
+ */
+export interface ModelContainer {
```

Comment:
This interface should be named ContainerDefinition instead of ModelContainer to be consistent with [sagemaker api](https://docs.aws.amazon.com/sagemaker/latest/APIReference/API_ContainerDefinition.html)
Also it should be replaced with [ContainerDefinition](https://github.com/aws/aws-cdk/pull/6107/files#diff-7d74ff670694308a90899138b87947e0R117) in sagemaker constructs when https://github.com/aws/aws-cdk/pull/6107 is merged to avoid code duplication